package.json

```diff
@@ -40,7 +39,7 @@
     "jsdom": "3.0.0",
     "jshint": "^2.6.3",
     "react": "^0.14.0",
-    "react-router": "^1.0.3",
+    "react-router": "^2.4.0",
```

Comment:
@samsel I've added the [fix](https://github.com/paypal/react-engine/pull/159/commits/8fd56aa95d9e2ec8ec60c4b542f6a6cce60e91bc) to #159.

I know the original PR was meant to update the example app, so would you like me to cherry-pick the fix and make a separate PR?